server/core/include/irods/notify_service_manager.hpp

```diff
@@ -0,0 +1,72 @@
+#ifndef IRODS_NOTIFY_SERVICE_MANAGER_HPP
+#define IRODS_NOTIFY_SERVICE_MANAGER_HPP
+
```

Comment:
May need a `/// \file` here for Doxygen to pick it up

Reply:
Added to both `notify_service_manager.hpp` and `chrono.hpp`.